misk-feature-testing/src/main/kotlin/misk/feature/testing/TestFeatureFlags.kt

```diff
@@ -0,0 +1,68 @@
+package misk.feature.testing
+
+import com.google.common.util.concurrent.AbstractIdleService
+import misk.feature.Attrs
+import misk.feature.Feature
+import misk.feature.FeatureFlags
+import misk.feature.FeatureService
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * In-memory test implementation of [FeatureFlags] that allows flags to be overridden.
```

Comment:
should this be thread-safe?

Reply:
Yeah. Should be fine to just make the underlying map a ConcurrentHashMap right?

Reply:
Yep!